### ISSUE TITLE
How to resize of QR code to be smaller in simple certificate?

### ISSUE THREAD

**Issue by williamtekkom** [bozoh/moodle-mod_simplecertificate#217]
![qr](https://user-images.githubusercontent.com/46547471/50940837-1fffba80-14b5-11e9-8ebc-d255f7fd6f5c.png)

**Comment by bozoh** [comment]
Only changing the source code.

https://github.com/bozoh/moodle-mod_simplecertificate/blob/master/locallib.php#L1093

`        $pdf->write2DBarcode($codeurl->out(false), 'QRCODE,M', $this->get_instance()->codex, $this->get_instance()->codey, 50, 50, .... `

the `50,50`  is the width and height  ,  if  the bottom text (code text) get weird,  comment this line:

https://github.com/bozoh/moodle-mod_simplecertificate/blob/master/locallib.php#L1097

**Comment by davefoord** [comment]
Knowing how to resize the QR code is really useful - but I also want to reposition it, so it appears bottom left rather than top left - how do I do this?

**Comment by anthonyandersondev** [comment]
> Knowing how to resize the QR code is really useful - but I also want to reposition it, so it appears bottom left rather than top left - how do I do this?

Hi Dave! 
Did you find how to do it?

**Comment by davefoord** [comment]
Yes - very easy to reposition, the settings are actually within each certificate activity, so you can have some bottom left, some top right, some in the middle etc. Within the certificate settings, and the 'Others Options' there is a 'show more' link at the top of the section, if you click on that, it gives you two boxes to identify the horizontal and vertical positions.

Note, you can also change the default settings at site admin level, for any new certificates.
